src/main/java/eu/europeana/metis/sandbox/service/dataset/DatasetService.java

```diff
@@ -26,7 +28,7 @@
    * @see Dataset
    */
   Dataset createDataset(String datasetName, Country country, Language language,
```

Comment:
We are breaking the contract here, while we could simply add another descriptive method and internally one would call the other to avoid duplications.
Not very important here because it's internal code, so I'll leave it to your own discretion.

Reply:
I don't quite understand what you mean, could you elaborate? Perhaps we can even do a quick call to clarify this

Reply:
I meant:
Keeping the old:
Dataset createDataset(String datasetName, Country country, Language language,
List<ByteArrayInputStream> records);

Creating a new:
Dataset createDataset(String datasetName, Country country, Language language,
List<ByteArrayInputStream> records, InputStream xsltTransformerEDMExternal);

And the old calls the new with an empty stream for the last parameter for backwards compatibility.
But as I said this is not hugely important since this code only resides in this project and it's not shared. 
So either way it would be fine.

Reply:
Ah now I understand. I see your point, and just like you said, for this specific case it's not really necessary since it's not a shared project, nevertheless I will add this, it's always good to improve the code quality whenever possible, and this could be useful in the future